src/main/java/proto/objects.proto

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
The proto file name(objects.proto) is not meaningful.  you can call it Pose.proto since it defines the pose proto.
Try to give descriptive file name and class name instead of generic name(usually the file name is the same as class name so it is obviously for users to know what the file is).